.github/workflows/go.yml

```diff
@@ -146,7 +146,7 @@ jobs:
         with:
           distribution: goreleaser-pro # either 'goreleaser' (default) or 'goreleaser-pro'
           # version: latest
-          args: release --split ${{ startsWith(github.ref, 'refs/tags/v') && '' || '--snapshot' }}
+          args: release --split ${{ !startsWith(github.ref, 'refs/tags/v') && '--snapshot' || '' }}
```

Comment:
Previously this always results in --snapshot because '' was interpreted as false. Use revert fix from @lionello . Duplicated below.